lib/generator._js

```diff
@@ -468,21 +428,24 @@ ScriptGenerator.prototype.generateDotNetDeploymentScript = function (templateFil
 ScriptGenerator.prototype.generateAspNetCoreScript = function (templateFileName, options, _) {
   argNotNull(templateFileName, 'templateFileName');
 
-  var fixedAspNetCoreProject = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.aspNetCoreProject) : fixPathSeperatorToUnix(options.aspNetCoreProject);
-  var fixedRestore = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.restore) : fixPathSeperatorToUnix(options.restore);
-  var fixedSolutionPath = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.solutionPath) : fixPathSeperatorToUnix(options.solutionPath);
-  
+  if(this.scriptType === scriptType.bash){
+      for(var prop in options){
+          options[prop] = fixPathSeperatorToUnix(options[prop])
+      }
+  }else{
```

Comment:
Try to keep the spacing consistent. e.g. should be `} else {`. A few other places.